src/index.ts

```diff
@@ -804,6 +815,13 @@ export function offlinePrecomputedInit(
     };
 
     shutdownEppoPrecomputedClient();
+
+    // Add memory-only assignment cache for offline mode
+    const assignmentCache = assignmentCacheFactory({
+      storageKeySuffix: 'offline',
+      forceMemoryOnly: true,
+    });
```

Comment:
Uses an in-memory assignment cache for offline initialization

Reply:
How come? Local storage or chome storage could still be "offline"?

Reply:
In-memory is consistent with how we do it in the `offlineInit` method

https://github.com/Eppo-exp/js-client-sdk/blob/dced1c2ded84a2173e6394542b16dd700da6931e/src/index.ts#L320-L327

I think it's a persistence vs access speed trade-off. The hybrid cache bridges the two goals, but won't work in the sync function due to the `assignmentCache.init` method being async. By choosing to go with an in-memory cache, we prioritized application speed over persistence across browser sessions and tabs, which makes sense since speed is likely also the priority of an SDK user who opts for an offline-init pattern

Reply:
> Local storage or chome storage

chrome storage is async and cannot be used here. although local storage is sync, it is so slow that I have been only wrappng it in async calls and I don't think its a good store to put into the execution path for evaluation.